DOCS/client-api-changes.rst

```diff
@@ -32,6 +32,10 @@ API changes
 
 ::
 
+ --- mpv 0.39.0 ---
+ 2.4    - mpv_render_param with MPV_RENDER_PARAM_ICC_PROFILE argument will no
```

Comment:
@sfan5 @Dudemanguy I'm not even sure, we should mention it in api changes. What do you think? It was really unusable without massive hackery on the API user side.

Reply:
I would write it as "no longer has broken allocation semantics and now works correctly"

Reply:
Yeah just "it's not broken" is good enough.

Reply:
I pushed the change, not sure if it is ok now.